app/src/test/java/org/mozilla/tv/firefox/utils/PreventLiveDataMainLooperCrashRule.kt

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.utils
+
+import android.arch.core.executor.testing.InstantTaskExecutorRule
+
+typealias PreventLiveDataMainLooperCrashRule = InstantTaskExecutorRule
```

Comment:
nit: kdoc. Why is this necessary?

Reply:
I think it's unclear what problem `@Rule val rule = InstantTaskExecutorRule()` is solving.  If someone is having trouble testing LiveData and looks back to one of these files, it will be immediately obvious how to fix it.  I don't know of any other situation in which we would use this rule, but if we run across one we can rename the alias.

Reply:
Maybe a separate issue but I wonder if we could subclass RobolectricTestRunner (or similar) and apply this rule to all tests.

Reply:
Probably.  I guess it couldn't really do any harm, although my knee-jerk reaction is that I like explicitly specifying custom test behavior where possible.

Reply:
I think the benefits are preventing violating DRY and making it clearer what is special about *this test* but we lose transparency.